Makefile

```diff
@@ -474,16 +474,14 @@ ci-job-tools: ci-setup-tools
 ### ci-runner-github-qemu jobs:
 
 define ci_setup_qemu_riscv
-	$(call banner,CI-Setup: Install Tock QEMU port)
+	$(call banner,CI-Setup: Build QEMU)
 	@# Use the latest QEMU as it has OpenTitan support
 	@printf "Building QEMU, this could take a few minutes\n\n"
-	# Download Tock qemu fork if needed
-	if ! bash -c 'cd tools/qemu && [[ $$(git rev-parse --short HEAD) == "7ff5b84" ]]'; then \
-		rm -rf tools/qemu; \
-		cd tools; git clone https://github.com/alistair23/qemu.git --depth 1 -b riscv-tock.next; \
+	@# Clone the lastest QEMU commit
+	@rm -rf tools/qemu; \
+		cd tools; git clone https://git.qemu.org/git/qemu.git --depth 1; \
```

Comment:
This needs to pin to a specific commit / version somehow (but I would wait until we've resolved the how before flipping around through more implementations).

Reply:
I would like to pin it to a SHA, but without submodules I don't see a great way to do this.